frontend/src/scenes/billing/v2/billingLogic.ts

```diff
@@ -59,6 +60,13 @@ export const billingLogic = kea<billingLogicType>([
             },
         ],
     }),
+    listeners({
+        loadBillingSuccess: ({ billing }) => {
+            if (billing.stripe_customer_id) {
+                posthog.group('customer', billing.stripe_customer_id)
+            }
+        },
+    }),
```

Comment:
Do we have to do this on the frontend? Seems like overkill to be sending the stripe_customer_id all the way through to the frontend just to track the group. Why don't we do this in the backend when we load from the billing service?